### PR TITLE
Fix another error that wasn't really an error

### PR DESCRIPTION
**Pull request by Arzte**
404 caused by user input and not related to the bot isn't an error, boi

### CODE REVIEW

File: Commands/Commands/queue.js [hunk 1]
```diff
@@ -607,16 +607,22 @@ function switchIDs (og, bot) {
 function deleteFromUV (UVID, uvClient, bot) {
   uvClient.v1.loginAsOwner().then(i => {
     i.delete(`forums/${config.uservoice.forumId}/suggestions/${UVID}.json`).catch((e) => {
+      if (e.statusCode === 404) {
+        bot.Channels.find(c => c.name === 'admin-queue').sendMessage('That suggestion doesn\'t seems to exist, (someone must have beat you too it).').then(o => {
+              setTimeout(() => bot.Messages.deleteMessages([o.id, msg.id], bot.Channels.find(c => c.name === 'admin-queue').id), config.timeouts.messageDelete)
```
Comment (Dougley) [inline Commands/Commands/queue.js]: - Supply the actual message objects, so you can leave the channel ID out.
- Format please.

**Review by Dougley — CHANGES_REQUESTED** [review]

File: Commands/Commands/queue.js [hunk 2]
```diff
@@ -607,16 +607,22 @@ function switchIDs (og, bot) {
 function deleteFromUV (UVID, uvClient, bot) {
   uvClient.v1.loginAsOwner().then(i => {
     i.delete(`forums/${config.uservoice.forumId}/suggestions/${UVID}.json`).catch((e) => {
+      if (e.statusCode === 404) {
+        bot.Channels.find(c => c.name === 'admin-queue').sendMessage('That suggestion doesn\'t seems to exist, (someone must have beat you too it).').then(o => {
```
Comment (LW001) [inline Commands/Commands/queue.js]: beat you **to** it pls

File: Commands/Commands/queue.js (hunk 1 above)
Comment (Arzte) [reply Commands/Commands/queue.js]: I was just copy pasting from another admin-queue reponse

File: Commands/Commands/queue.js [hunk 3]
```diff
@@ -604,23 +604,29 @@ function switchIDs (og, bot) {
   })
 }
 
-function deleteFromUV (UVID, uvClient, bot) {
+function deleteFromUV(UVID, uvClient, bot) {
   uvClient.v1.loginAsOwner().then(i => {
     i.delete(`forums/${config.uservoice.forumId}/suggestions/${UVID}.json`).catch((e) => {
+      if (e.statusCode === 404) {
+        bot.Channels.find(c => c.name === 'admin-queue').sendMessage('That suggestion doesn\'t seems to exist, (someone must have beat you to it).').then(o => {
```
Comment (Dougley) [inline Commands/Commands/queue.js]: `doesn't seem to`

**Review by Dougley — CHANGES_REQUESTED** [review]

**Comment by Arzte** [comment]
Unfortunately with all the work on master and me not being at my main machine (on laptop) I had to redo this pr without the formatting fixes. Otherwise this should be good to land in master.

File: Commands/Commands/queue.js [hunk 4]
```diff
@@ -521,14 +521,20 @@ function merge (target, dupe, uv) {
 function deleteFromUV (UVID, uvClient, bot) {
   uvClient.v1.loginAsOwner().then(i => {
     i.delete(`forums/${config.uservoice.forumId}/suggestions/${UVID}.json`).catch((e) => {
-      logger.log(bot, {
-        cause: 'card_destroy',
-        message: (e.message !== undefined) ? e.message : JSON.stringify(e)
-      }, e)
+        if (e.statusCode === 404) {
+          bot.Channels.find(c => c.name === 'admin-queue').sendMessage('That suggestion doesn\'t seem to exist, (someone must have beat you to it).').then(o => {
+              setTimeout() => o.delete()
```
Comment (Dougley) [inline Commands/Commands/queue.js]: Syntax error

File: Commands/Commands/queue.js [hunk 5]
```diff
@@ -521,14 +521,20 @@ function merge (target, dupe, uv) {
...
+          })
+        } else {
+            logger.log(bot, {
```
Comment (Dougley) [inline Commands/Commands/queue.js]: Indent using 2 spaces

**Review by Dougley — CHANGES_REQUESTED** [review]

File: Commands/Commands/queue.js (hunk 4 above)
Comment (Arzte) [reply Commands/Commands/queue.js]: Where?

File: Commands/Commands/queue.js (hunk 5 above)
Comment (Arzte) [reply Commands/Commands/queue.js]: Looks awful, but ok

File: Commands/Commands/queue.js (hunk 4 above)
Comment (Dougley) [reply Commands/Commands/queue.js]: `setTimeout(() => o.delete(), config.timeouts.whatever)`

**Comment by Arzte** [comment]
Yo, I fixed the two things you pointed out, it good now?

**Pull request merged**